client/services.go

```diff
@@ -598,3 +599,9 @@ type WorkspacesClient interface {
 	DescribeWorkspaces(ctx context.Context, params *workspaces.DescribeWorkspacesInput, optFns ...func(*workspaces.Options)) (*workspaces.DescribeWorkspacesOutput, error)
 	DescribeWorkspaceDirectories(ctx context.Context, params *workspaces.DescribeWorkspaceDirectoriesInput, optFns ...func(*workspaces.Options)) (*workspaces.DescribeWorkspaceDirectoriesOutput, error)
 }
+
+//go:generate mockgen -package=mocks -destination=./mocks/mock_codepipeline.go . CodePipelineClient
+type CodePipelineClient interface {
```

Comment:
would maybe add https://docs.aws.amazon.com/codepipeline/latest/APIReference/API_ListWebhooks.html API, would let @bbernays decide if we should do this in this PR or another issue.

Reply:
We can do it in another PR. I have opened an issue to track [that work:](https://github.com/cloudquery/cq-provider-aws/issues/658)